README.txt

```diff
@@ -40,9 +40,9 @@ When using OAuth, Github expects the OAuth token to be presented as your
 username, and your password should either be blank or set to 'x-oauth-basic'.
 These will need to be set, in addition to your Github username.  For example:
 
-    git config --global omnifocus-github.user 51951ceb3819276195e8525740bd4670232bc222
-    git config --global omnifocus-github.password x-oauth-basic
-    git config --global omnifocus-github.name me
```

Comment:
So, I think this is confusing also because it isn't technically correct. This is in the section of the readme for supporting multiple accounts ala Github Enterprise. Since that's the edgecase and not the norm, I think it needs to have its own section. Can you dupe and strip it down to the minimum default case?